### ISSUE TITLE
loss_D in SRGAN 

### ISSUE THREAD

**Issue by alsombra** [eriklindernoren/PyTorch-GAN#43]
Hello and my thanks for this great repo. I like how your code is simple and effective.

I would like to point out that you are using MSE for your Discriminator Loss instead of Binary Cross Entropy. If you have a specific reason for why you are doing that, could you share it?

**Comment by eriklindernoren** [comment]
Hi, thanks. :) Yes, I use a least squares objective for the discriminator. This way of training GANs was first introduced in the paper on LSGAN (https://arxiv.org/abs/1611.04076), and has been shown to help solve convergence issues for GANs. It's pretty common on more recent GAN varieties and I have found it to work quite well.

**Comment by alsombra** [comment]
Thanks for the reply and for the source paper. I did not know that. Indeed, I tried training your srgan using the both BCE and MSE losses just to make sure and the results looked pretty similar!

**Comment by HareshMiriyala** [comment]
this is the same case with the conditional gan example.  Thanks for clarifying.
